lib/structures/Message.js

```diff
@@ -103,7 +121,10 @@ class Message extends Base {
             data.content = `${this.author.mention} just boosted the server!`;
         } else if(this.type === 9 || this.type === 10 || this.type === 11) {
             data.content = `${this.author.mention} just boosted the server! ${this.channel.guild ? this.channel.guild.name : data.guild_id} has achieved **Level ${this.type - 8}!**`;
-        }  else {
+        } else if(this.type === 12) {
+            const guild = data.content.split('#');
+            data.content = `${this.author.mention} has added ${guild.length && guild[0]}<#${this.messageReference && this.messageReference.channelID}> to this channel`;
```

Comment:
The channel ID is not going to resolve for many people, think it would be better to just use `data.content` here instead of the whole `guildName+channelID` thing, and keep ID accessible for bots from `messageReference`